content/docs/reference/pricing-and-costs.md

```diff
@@ -46,8 +46,5 @@ For a FREE @sign there exists an @sign generator which creates a new set of @sig
     * Modest89rackets
     * 90bossaball
 
-<!-- TODO: This button should link to the get an @sign page -->
-</div>
-<center>
-{{< button/link href="#" type="primary" text="Get started for free" >}}
-</center>
\ No newline at end of file
+<!-- TODO: This button should link to the get an @sign page and figure out a shortcode that centers this.git -->
```

Comment:
Remove the todo as well please

Reply:
Done this on commit https://github.com/atsign-foundation/atsign.dev-3.0/pull/54/commits/e4dc966b88b3adf9b0a8d622d6ae41cb019a384f